Formula/mongodb-community-shell.rb

```diff
@@ -1,8 +1,30 @@
 class MongodbCommunityShell < Formula
   desc "An interactive JavaScript command-line interface to MongoDB"
   homepage "https://www.mongodb.com/"
-  url "https://fastdl.mongodb.org/osx/mongodb-shell-osx-ssl-x86_64-4.0.6.tgz"
-  sha256 "bcc6c3da65dcdb62b09b044a7f578e0caada5ac1051f6ef41456bc517a542b6a"
+
+  # frozen_string_literal: true
```

Comment:
remove commented code

Reply:
rubocop wanted that, not me. :-) I can remove it if you want though (the main formula also has it since the linter wanted it there).

Reply:
I'm dumb.